kuksa-client/kuksa_client/__init__.py

```diff
@@ -86,17 +87,23 @@ def getValues(self, paths: Iterable[str], attribute="value", timeout=5):
     # The given callback function will be called then, if the given path is updated:
     #   updateMessage = await webSocket.recv()
     #   callback(updateMessage)
-    def subscribe(self, path, callback, attribute = "value", timeout=5):
+    def subscribe(self, path, callback, attribute="value", timeout=5):
         return self.backend.subscribe(path, callback, attribute, timeout)
 
-    def subscribeMultiple(self, paths, callback, attribute = "value", timeout=5):
+    def subscribeMultiple(self, paths, callback, attribute="value", timeout=5):
         return self.backend.subscribeMultiple(paths, callback, attribute, timeout)
 
     # Unsubscribe value changes of to a given path.
     # The subscription id from the response of the corresponding subscription request will be required
     def unsubscribe(self, sub_id, timeout=5):
         return self.backend.unsubscribe(sub_id, timeout)
 
+    def disconnect(self, timeout=5):
+        return self.backend.disconnect(timeout)
+
+    def connect(self, timeout=5):
```

Comment:
Wouln't this break all existing users of threaded API, i.e. GPS feeder, S3 example etc? Because they are not calling connect?

Reply:
I don't think so because they all call connect from within ```__enter__``` and the threaded API works with the cli_backends which leverage asysnc/sync library parts. I tested with gps feeder and a virtual environment and worked for me.